main.scss

```diff
@@ -19,8 +19,8 @@
 		height: var(--o-meter-height);
 		--o-meter-background-color: #{oColorsByName('slate-white-15')};
 		--o-meter-optimum-color: #{oColorsByName('jade')};
-		--o-meter-low-color: #{oColorsByName('crimson')}; //bad
-		--o-meter-high-color: #ff8833; // 'mandarin' color used as hex because isn't an internal brand color
+		--o-meter-sub-optimum-color: #ff8833; // 'mandarin' color used as hex because isn't an internal brand color
```

Comment:
I know this isn't part of this change, but should we add Mandarin to the internal brand?

Reply:
Maybe, I'm not sure it deserves to be available to all to support this one usecase, unless the design team show an intent to use it more widely within the internal brand

Reply:
https://github.com/Financial-Times/o-colors/issues/245